models/POP/shell_scripts/cesm2_1/CESM2_1_setup_pmo

```diff
@@ -0,0 +1,763 @@
+./xmlchange --subgroup case.run --id JOB_WALLCLOCK_TIME --val $timewall
+
+# ==============================================================================
+# Setting up the case.
+#
+# This creates the EXEROOT and RUNDIR directories.
+# ==============================================================================
+
+echo ''
+echo 'Setting up the case ...'
+echo 'This creates the EXEROOT and RUNDIR directories.'
+echo ''
+
+./case.setup
+
+if ( $status != 0 ) then
+   echo "ERROR: Case could not be set up."
+   exit -2
+endif
+
+# ==============================================================================
+# Modifying namelist templates for each instance.
+# ==============================================================================
+
+echo ''
+echo 'Modifying namelist templates for each instance.'
+echo ''
+
+@ inst = 1
+while ($inst <= 1)
+
+   # following the CESM strategy for 'inst_string'
+   set inst_string = ''
+
+   # ===========================================================================
+   set fname = "user_nl_datm${inst_string}"
+   # ===========================================================================
+   # DATM Namelist
+
+   echo "dtlimit  = 1.5, 1.5"               >> ${fname}
+   echo "fillalgo = 'nn', 'nn'"             >> ${fname}
+   echo "fillmask = 'nomask','nomask'"      >> ${fname}
+   echo "mapalgo  = 'bilinear','bilinear'"  >> ${fname}
+   echo "mapmask  = 'nomask','nomask'"      >> ${fname}
+   echo "streams  = 'datm.streams.txt.CPLHISTForcing.nonSolarFlux${inst_string} $stream_year_align $stream_year_first $stream_year_last'," >> ${fname}
+   echo "           'datm.streams.txt.CPLHISTForcing.Solar${inst_string}        $stream_year_align $stream_year_first $stream_year_last',"  >> ${fname}
+   echo "           'datm.streams.txt.CPLHISTForcing.State1hr${inst_string}     $stream_year_align $stream_year_first $stream_year_last',"  >> ${fname}
+   echo "           'datm.streams.txt.CPLHISTForcing.State3hr${inst_string}     $stream_year_align $stream_year_first $stream_year_last'"  >> ${fname}
+   echo "taxmode  = 'cycle','cycle'"        >> ${fname}
+   echo "tintalgo = 'linear','linear'"      >> ${fname}
+   echo "restfils = 'unset'"                >> ${fname}
+   echo "restfilm = 'unset'"                >> ${fname}
+
+   # ===========================================================================
+   set fname = "user_nl_cice${inst_string}"
+   # ===========================================================================
+   # CICE Namelist
+   # Note well: this is true as of CESM2_1 but may change in the future.
+   # In a hybrid case, setting restart files for CICE is a distinct process from
+   # setting restart files for POP. The restart file is set in the user_nl_cice
+   # file in the ${CASEROOT} directory using the ice_ic (ice initial condition)
+   # key in the namelist file instead of using an rpointer file in ${RUNDIR}.
```

Comment:
Edited out that clause so the section reads `In a hybrid case, setting restart files for CICE is set in the user_nl_cice file in the ${CASEROOT} directory using the ice_ic (ice initial condition) key in the namelist file instead of using an rpointer file in ${RUNDIR}.